src/core/Entry.cpp

```diff
@@ -916,11 +917,83 @@ QString Entry::resolvePlaceholderRecursive(const QString& placeholder, int maxDe
     }
     case PlaceholderType::Reference:
         return resolveReferencePlaceholderRecursive(placeholder, maxDepth);
+    case PlaceholderType::DateTimeSimple:
+    case PlaceholderType::DateTimeYear:
+    case PlaceholderType::DateTimeMonth:
+    case PlaceholderType::DateTimeDay:
+    case PlaceholderType::DateTimeHour:
+    case PlaceholderType::DateTimeMinute:
+    case PlaceholderType::DateTimeSecond:
+    case PlaceholderType::DateTimeUtcSimple:
+    case PlaceholderType::DateTimeUtcYear:
+    case PlaceholderType::DateTimeUtcMonth:
+    case PlaceholderType::DateTimeUtcDay:
+    case PlaceholderType::DateTimeUtcHour:
+    case PlaceholderType::DateTimeUtcMinute:
+    case PlaceholderType::DateTimeUtcSecond:
+        return resolveMultiplePlaceholdersRecursive(resolveDateTimePlaceholder(typeOfPlaceholder), maxDepth - 1);
     }
 
     return placeholder;
 }
 
+QString Entry::resolveDateTimePlaceholder(Entry::PlaceholderType placeholderType) const
+{
+    time_t time = std::time(NULL);
+    char date_formatted[50]{};
+
+    switch (placeholderType) {
+    case PlaceholderType::DateTimeSimple:
+        std::strftime(date_formatted, sizeof(date_formatted), "%Y%m%d%H%M", localtime(&time));
```

Comment:
I would prefer you use the Qt datetime classes.

Reply:
Use `Clock` class

Reply:
I do now, using `QDateTime::toString`.